CMakeLists.txt

```diff
@@ -77,7 +84,7 @@ endif()
 set_property(TARGET stem PROPERTY POSITION_INDEPENDENT_CODE ON)
 
 target_link_libraries(stem
-  PRIVATE ${_libs})
+  PRIVATE "${_libs}")
```

Comment:
Why the quotes here? Lists should expand as expected without them - is there an issue with HDF5 needing this?

Reply:
Not sure why, I removed them and it work fine.